rbac/management/tenant_service/v2.py

```diff
@@ -114,12 +203,13 @@ def update_users(self, users: list[User]):
         Args:
             users (list): List of User objects to update
         """
-        bootstrapped_list = []
+        org_ids = []
         for user in users:
             if user.org_id is None:
                 logger.warning(f"Cannot update user without org_id. Skipping. username={user.username}")
                 continue
```

Comment:
I guess this assumes all users are enabled. We should probably just add a guard around this similar to the org_id check. e.g. 

```suggestion
            if not user.is_active:
                continue
            if user.org_id is None:
                logger.warning(f"Cannot update user without org_id. Skipping. username={user.username}")
                continue
```

(not sure if worth warning since there's no issue really if the user is not active)

Reply:
Although, we'd also need to check below (loop at line 227)

Reply:
Yea, assuming all users enabled. Added the guard